### PR TITLE
Allow enabling and disabling of blocks for generator flowgraph

### PR DESCRIPTION
**Pull request by reiinakano**
generatorr grc

Same thing as the previous PR. Wrapped some blocks in hierarchical blocks so we can enable and disable at runtime.

### CODE REVIEW

File: gr-starcoder_utils/python/rs_encoder.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    gr-satellites. It adds an additional parameter `enabled` to determine
+    whether to decode the packet or let it through without change.
+    """
+    def __init__(self, basis, enabled):
+        gr.hier_block2.__init__(self,
+            "rs_encoder",
+            gr.io_signature(0, 0, 0),  # Input signature
+            gr.io_signature(0, 0, 0))  # Output signature
+
+        self.log = gr.logger("log")
```
Comment (kenichi-fukushima) [inline gr-starcoder_utils/python/rs_encoder.py]: not used

File: gr-starcoder_utils/python/conv_encoder.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+# the Free Software Foundation, Inc., 51 Franklin Street,
+# Boston, MA 02110-1301, USA.
+# 
+
+from gnuradio import gr
+from gnuradio import fec
+from pmt_pass import pmt_pass
+
+
+class conv_encoder(gr.hier_block2):
```
Comment (kenichi-fukushima) [inline gr-starcoder_utils/python/conv_encoder.py]: Don't abbreviate unless this is a well-established convention in GnuRadio or DSP in general. I can't tell whether `conv` means conversion or convolution.

Also why isn't it named after the wrapped block?

**Comment by reiinakano** [comment]
> I started to suspect you might use some meta-programming technique to generate these simple wrappers.

Let me think about this. It's not very hard to do in Python, but I'm not sure how to make sure we can extend this to GNURadio Companion. Let me know if you have any suggestions.

**Comment by reiinakano** [comment]
Closing, I think I thought of a way to make this generic.

**Pull request closed without merging**